tests/phpunit/Process/Runner/MutationTestingRunnerTest.php

```diff
@@ -183,23 +183,28 @@ public function test_it_dispatches_events_even_when_no_mutations_is_given(): voi
         $threadCount = 4;
         $testFrameworkExtraOptions = '--filter=acme/FooTest.php';
 
+        $this->processBuilderMock
```

Comment:
My understanding was that it was the default behaviour of mocks wasn't it?

Reply:
I'm 95% sure that unconfigured mocks instead will allow any number of calls on all methods, and they try to return a 'default' value.

Reply:
then can we check? Because I removed it elsewhere as well so that would require to revert that + IIRC there was mention of a `$this->createStub()` in the new versions which would help with that as well

Reply:
I've tested it, and my assumption was correct about the default behaviour of phpunit mocks:
The follow test code passes.
```php
<?php

namespace Fooa;

use PHPUnit\Framework\TestCase;

class A{
    public function foo(){}
}

class B{
    /**
     * @var A
     */
    private $a;

    public function __construct(A $a)
    {
        $this->a = $a;
    }

    public function foo(){$this->a->foo();}
}

class ATest extends TestCase
{
    public function testMock(): void
    {
        $a = $this->createMock(A::class);

        $b = new B($a);
        $b->foo();
        $b->foo();
        $b->foo();
        $b->foo();
        $b->foo();
        $this->assertTrue(true);

    }
}

```

Reply:
I'll try to double check our codebase then because we most likely have that issue in other places